zerver/lib/slack_data_to_zulip_data.py

```diff
@@ -27,23 +28,31 @@ def rm_tree(path: str) -> None:
     if os.path.exists(path):
         shutil.rmtree(path)
 
-def get_model_id(model: Any, table_name: str, sequence_increase_factor: int) -> int:
+def idseq(model_class: Any) -> str:
+    return '{}_id_seq'.format(model_class._meta.db_table)
+
+def get_next_id(model_class: Any) -> int:
+    conn = connection.cursor()
+    sequence = idseq(model_class)
+    conn.execute("select nextval('%s')" % (sequence))
+    row = conn.fetchone()
+    conn.close()
+    return row[0]
+
+def reset_sequence(model_class: Any, value: int) -> None:
+    conn = connection.cursor()
+    sequence = idseq(model_class)
+    conn.execute("ALTER SEQUENCE {} RESTART WITH {};".format(sequence, value))
+
+def allocate_id(model_class: Any, sequence_increase_factor: int) -> int:
```

Comment:
How much is the perf improvement of this new method?

Reply:
Test now takes less than 1 second.

Reply:
cool